drydock/plugin.py

```diff
@@ -15,6 +147,7 @@
         "INIT_JOBS": False,
         "CMS_SSO_USER": "cms",
         "AUTO_TLS": True,
+        "MIGRATE_FROM": "",
```

Comment:
```suggestion
        "MIGRATE_FROM": 0,
```
Being an empty string it gives the idea it should be populated with a string (but it is an int)